### PR TITLE
Drop all stale migrations

### PR DESCRIPTION
**Pull request by NunoAlexandre**
# Description

Drop migration code for a few pallets that should have already been executed.

## Type of change

- Deleting stale code

### CODE REVIEW

**Review by mikiquantum — COMMENTED** [review]
LGTM, one question, there is a migration in the Altair runtime, that could be removed as well right? `pallet_pool_system`. Since this is going live soon.

**Comment by NunoAlexandre** [comment]
@branan can you please have a look as well? We also delete the `pool_system` migrations so it's good to get your approval here

File: pallets/interest-accrual/src/lib.rs
Comment (mustermeiszer) [inline pallets/interest-accrual/src/lib.rs:420]: I think `remove_unused_rates` was also part of the migration and can be dropped.

**Review by mustermeiszer — COMMENTED** [review]
LGTM! Thanks for handling. There is one function in interest accrual that I think can be removed, too. Cc: @branan 

And am not approving as I was not involved in anchors.

File: pallets/interest-accrual/src/lib.rs
Comment (NunoAlexandre) [reply pallets/interest-accrual/src/lib.rs:420]: It seems unused, dropped it 👍

Comment (NunoAlexandre) [reply pallets/interest-accrual/src/lib.rs:420]: @branan appreciate your confirmation here

Comment (branan) [reply pallets/interest-accrual/src/lib.rs:420]: Yup, that was part of the migration (a nasty back-and-forth of data between interest-accrual and loans). Kill it with fire

Comment (branan) [reply pallets/interest-accrual/src/lib.rs:420]: There is also a `reference_active_rates` in pallet-loans that can go bye-bye now, which was part of the same dance

Comment (NunoAlexandre) [reply pallets/interest-accrual/src/lib.rs:420]: thanks @branan, done 👍

**Review by mustermeiszer — APPROVED** [review]
Approving after everybody went through!

**Comment by NunoAlexandre** [comment]
> Approving after everybody went through!

Thanks @mustermeiszer, a true gentleman 🤘🏻

**Pull request merged**
